plugin/geoip/city.go

```diff
@@ -6,27 +6,25 @@ import (
 
 	"github.com/coredns/coredns/plugin/metadata"
 
-	"github.com/oschwald/geoip2-golang"
+	"github.com/oschwald/geoip2-golang/v2"
 )
 
-const defaultLang = "en"
```

Comment:
I like the idea of keeping a defaultLang set to English, to eventually allow users to configure their language of choice.